elasticsearch-model/lib/elasticsearch/model/adapters/mongoid.rb

```diff
@@ -64,9 +64,16 @@ module Importing
           #
           def __find_in_batches(options={}, &block)
             options[:batch_size] ||= 1_000
+            query = options.delete(:query)
+            named_scope = options.delete(:scope)
+            preprocess = options.delete(:preprocess)
```

Comment:
Do you mind if we just set the `:query`, `:scope` and `:preprocess` options instead of deleting them from the `options` hash? I'd rather not mutate the `options` that the user passes to the `#import` method. Thanks!